src/Tribe/Ticket_Object.php

```diff
@@ -1149,6 +1151,23 @@ public function has_meta_enabled() {
 			 */
 			return (bool) apply_filters( 'tribe_tickets_has_meta_enabled', false, $this->ID );
 		}
+
+		/**
+		 * Determine whether the ticket has ticket cache enabled.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool Whether the ticket has ticket cache enabled.
+		 */
+		public function is_ticket_cache_enabled() {
+			/**
+			 * Allow filtering whether the ticket has ticket cache enabled.
+			 *
```

Comment:
Let's add a `@since` to the filter as well.

Reply:
```suggestion
			 *
			 * @since TBD
			 *
```